src/session.cpp

```diff
@@ -328,7 +335,7 @@ session_t::memory_pressure() const {
 
 std::string
 session_t::name() const {
```

Comment:
So that would make logging of such return values a mess. We'd either end up doing something like `session.name().get_value_or("<none>")` all the same or writing a separate `COCAINE_LOG_*` for each case.